### PR TITLE
support deleting tweets when posts are moderated

### PR DESCRIPTION
**Pull request by WebFreak001**
Twitter API usage untested, but moderation sink seems to work.

### CODE REVIEW

File: src/dfeed/sinks/twitter.d [hunk 1]
```diff
@@ -84,10 +88,70 @@ protected:
 		request.method = "POST";
 		request.headers["Authorization"] = session.prepareRequest(baseURL, "POST", parameters).oauthHeader;
 		request.data = DataVec(Data([]));
-		httpRequest(request, null);
+		httpRequest(request, delegate(HttpResponse response, string disconnectReason) {
+			if (response)
+			{
+				struct ResponseT
```
Comment (CyberShadow) [inline src/dfeed/sinks/twitter.d:94]:
```suggestion
				@JSONPartial
				struct ResponseT
```

**Review by CyberShadow — COMMENTED** [review]
Nice

**Comment by CyberShadow** [comment]
Would you mind rebasing to fix CI?

**Comment by WebFreak001** [comment]
it looks like I'm already up-to-date

**Comment by WebFreak001** [comment]
done

**Comment by CyberShadow** [comment]
GitHub seems to be having some network issues right now

**Comment by CyberShadow** [comment]
OK, it's deployed.

**Pull request merged**